tox.ini

```diff
@@ -59,20 +59,38 @@ commands =
     python --version
     pip-audit {env:EXCLUDES:}
 
-[testenv:docs,docs-py{310,311,312}]
+[testenv:docs]
```

Comment:
Why did you remove the option to specify the python version?

Reply:
see PR description